modules/channel-web/src/views/lite/store/index.ts

```diff
@@ -427,20 +426,21 @@ class RootStore {
     this.view.disableAnimations = this.config.disableAnimations
     this.config.showPoweredBy ? this.view.showPoweredBy() : this.view.hidePoweredBy()
 
-    const locale = getUserLocale(this.config.locale)
-    this.config.locale && this.updateBotUILanguage(locale)
-    document.documentElement.setAttribute('lang', locale)
-
     document.title = this.config.botName || 'Botpress Webchat'
 
-    try {
+    this.api.updateAxiosConfig({ botId: this.config.botId, externalAuthToken: this.config.externalAuthToken })
+    this.api.updateUserId(this.config.userId)
+
+    if (!this.isInitialized) {
       window.USE_SESSION_STORAGE = this.config.useSessionStorage
-    } catch {
-      console.error('Could not set USE_SESSION_STORAGE')
+    } else {
+      console.warn('[WebChat] "useSessionStorage" value cannot be altered once the webchat is initialized')
```

Comment:
Won't this logic just always print the warning even if you didnt change the property ? shouldnt it be like
```js
if(this.isInitialized && window.USE_SESSION_STORAGE !== this.config.useSessionStorage){
 // warn
}```

Reply:
Right, good point!